src/Component/Dash.js

```diff
@@ -0,0 +1,309 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+import { GoogleLogout } from 'react-google-login';
+import { useNavigate } from 'react-router-dom';
+import { Button } from '@material-ui/core';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Drawer from '@material-ui/core/Drawer';
+import AddBoxIcon from '@mui/icons-material/AddBox';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import List from '@material-ui/core/List';
+import Container from '@material-ui/core/Container';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import Box from '@mui/material/Box';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Paper from '@mui/material/Paper';
+
+import TextField from '@material-ui/core/TextField';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import DatePicker from '@mui/lab/DatePicker';
+
+import CurLeaves from './CurLeaves';
+import PerticularLeaves from './PerticularLeaves';
+import Title from './Title';
+import AddLeaveDialog from './AddLeaveDialog';
+
+const drawerWidth = 240;
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+  },
+  toolbar: {
```

Comment:
Should we keep styles in another file?

Reply:
Yes, We can.. But toolbarIcon: { display: 'flex', alignItems: 'center', justifyContent: 'flex-end', padding: '0 8px', ...theme.mixins.toolbar, }, This type of component give error.. & i use this template from MUI so i don't know how to move this css into other file.

Reply:
Can we create a separate styles(css/scss) file?